moveit_core/distance_field/src/propagation_distance_field.cpp

```diff
@@ -226,11 +226,11 @@ void PropagationDistanceField::removePointsFromField(const EigenSTL::vector_Vect
   removeObstacleVoxels(voxel_points);
 }
 
-void PropagationDistanceField::addNewObstacleVoxels(const std::vector<Eigen::Vector3i>& voxel_points)
+void PropagationDistanceField::addNewObstacleVoxels(const StlVectorEigenVector3i &voxel_points)
```

Comment:
I'm unsure about how allocators are handled, but I guess these changes break API?
Even then, Eigen alignment issues are critical bugs and there's no other way to fix them.
So let's go ahead with the patch.